### ISSUE TITLE
Unable to scroll results window intermittently

### ISSUE THREAD

**Issue by jakewarren** [center-for-threat-informed-defense/attack-powered-suit#7]
**Describe the bug**

Intermittently I am unable to scroll the results window when multiple results are returned for a particular search. Unfortunately I am not able to figure out what is causing it, not getting any error messages in the extension devtools console either. Sometimes I am able to scroll through the results as expected and sometimes it doesn't allow scrolling. Using the devtools I can see the results in the DOM so I do not believe it is an issue with the search results. 

**To Reproduce**

Note: due to the random nature of the issue it may take multiple attempts to replicate.

1. Open the extension pop-up
2. Enter a search term such as `denial` or `tunnel` that is expected to have many results
3. Attempt to scroll through the search results

**Platform:**
 - OS: MacOS
 - Browser: Google Chrome
 - Browser Version: 103.0.5060.53
 - Extension Version: 1.0.1

> If you want to express positive or negative feedback on a bug report, please leave a Thumbs Up or Thumbs Down so that we can keep track of demand. Please do not reply just to say +1 or -1. Thanks!

**Comment by mehaase** [comment]
Thanks for reporting this. How frequently does this happen? I tried 10 times in a row and could not trigger it.

If anybody else is experiencing this, please respond to @jakewarren's top post with the 👍 emoji so I can track.

**Comment by jakewarren** [comment]
@mehaase it happens more often than not. In an attempt to rule out a conflict with another extension I created a new profile in Chrome and installed the extension there. I created a screen recording to help illustrate the issue of not being able to scroll down to reach the copy shortcuts; unfortunately I don't know of software that can visually depict scrolling in a screen recording so I just wiggled the mouse to indicate that I was attempting to scroll. 

I tested the extension in Brave and it works flawlessly for me; so not sure what's going on. 

https://user-images.githubusercontent.com/6472686/176698076-4a190b1e-a359-4f07-823a-767da75a0f79.mov
